### DISCUSSION TITLE
An error occured: cannot acces events anymore

### DISCUSSION THREAD

**Discussion by JanineKall** [liedekef/events-made-easy#447] [Q&A] [answered]
Hi, i was testing new features in a test event. I probably did something wrong, and now I cannot access any of my events. I get the message: 
![image](https://github.com/liedekef/events-made-easy/assets/129589814/46bf80b9-a52c-406e-931d-0a37a1b31a06)
I don't know what to do now. Can you please help?

**Comment by liedekef** [comment]
Based on this, the server returned an unexpected answer, causing this to display. This is not necessarily EME's fault, but based on this screenshot there's nothing more I can say. You should check your browser console for what is being returned by the server.

**Comment by JanineKall** [comment]
Hi Franky,

 

Thanks fort he reply.

I really think it’s caused by something I did wrong in EME. When I filter the events, I can access all events accept the test event which I was editing. When I try to access that particular event ánd when I want to display all (future) events, I get the message. 

 

I checked my browser console and combined some screenshots in one:



 

Does this help?

Thanks!

 

ir. Janine Kallenbach - Heinen

**Comment by liedekef** [comment]
I don't see any screenshots (I advise you not to reply via mail but via the github site)

**Comment by JanineKall** [comment]
![image](https://github.com/liedekef/events-made-easy/assets/129589814/469e85c7-f6f6-4b16-ba8d-43659274285d)
;-)

**Reply by liedekef** [reply]
Those are irrelevant and unrelated to your issue. The issue is when doing the "filter" action, so when the server replies.
If unclear, feel free to send me a temp admin account with detailed instructions on how to reproduce the issue.

**Reply by JanineKall** [reply]
I'm sorry, doing my best ;-)
I'll send you an admin invite. To the gmail account?

**Reply by liedekef** [reply]
yes

**Reply by JanineKall** [reply]
Done.

**Reply by liedekef** [reply]
You have PHP errors on your site. Check your PHP errorlog for info (not visible in EME)

**Reply by JanineKall** [reply]
Ok I'll check with the host. Unfortunately I don't have hosting access to this site. ;-(

**Reply by liedekef** [reply]
I'll install "Query Monitor" on your site and see if that gives more info (sometimes it does) and remove it afterwards. Next to that: you don't seem to have a security plugin installed, that might need to change :-)

**Reply by liedekef** [reply]
Next to that: please upgrade EME to the latest version, 2.3.14 is out of date. See the main github site of EME: https://github.com/liedekef/events-made-easy/ and search for "For existing wordpress users that have version 2.3.18 or older" ==> upgrading EME might already solve the problem.

**Reply by JanineKall** [reply]
I noticed you upgraded EME but the error stil occurs. I also noticed you ran Query Monitor. Did anything come up?

**Reply by liedekef** [reply]
I did not upgrade EME, I leave that up to you :-) And Query Monitor just showed me that there was a php issue, but I'm guessing because EME is too old (php incomptibilty probably: 2.3.14 is from 2022/12/14)
So upgrade EME first and see what gives.

**Reply by liedekef** [reply]
If you feel unsure about the upgrade: I can do that (but not free, that doesn't belong to EME support in general)

**Reply by JanineKall** [reply]
Thing is it's not up to me to do that, we contracted someone to do that. So I'll urge him to do so.

**Reply by JanineKall** [reply]
He's quick; he's on it.

**Reply by JanineKall** [reply, marked as answer]
He upgraded EME and the error is gone. Pfiew. Thanks so much!

**Reply by liedekef** [reply]
Good to hear!
